gcloud-java-pubsub/src/main/java/com/google/gcloud/pubsub/spi/DefaultPubSubRpc.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.pubsub.spi;
+
+import com.google.gcloud.pubsub.PubSubOptions;
+
+public class DefaultPubSubRpc implements PubSubRpc {
+
+  private final PubSubOptions options;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'options' could be replaced by a local variable.](https://www.codacy.com/app/mziccard/gcloud-java/file/2296608824/issues/source?bid=3123109&fileBranchId=3188953#l23)